access/client.go

```diff
@@ -0,0 +1,261 @@
+package access
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/safing/portbase/api"
+	"github.com/safing/portbase/database"
+	"github.com/safing/portbase/formats/dsd"
+	"github.com/safing/portbase/log"
+	"github.com/safing/spn/access/account"
+)
+
+// https://account.safing.io/v1/authenticate
+
+const (
+	AccountServer   = "https://account.safing.io"
+	LoginPath       = "/v1/authenticate"
+	UserProfilePath = "/v1/user_profile"
+)
```

Comment:
If we want to add new user related APIs later I would recommend to go with `/v1/user/profile` instead.